### PR TITLE
Don't unnecessarily reset y domain on line charts

### LINKED ISSUE #8994: Reload Button in Tensorboard 1.1rc1 Resets Chart Scales
In tensorboard 1.1rc1, hitting the "reload button",  image, causes the scaling on the charts to reset. That was not the case in 1.0.1.

For example if I zoom in here:
image
to:
image
this zoom is lost.

### PR DESCRIPTION
**Pull request by taion**
Fixes #8994

### CODE REVIEW

File: tensorflow/tensorboard/components/vz_line_chart/dragZoomInteraction.ts [hunk 1]
```diff
@@ -141,21 +140,14 @@ export class DragZoomLayer extends Plottable.Components.SelectionBoxLayer {
...
   // Restore the scales to their state before any zoom
   private unzoom() {
-    if (!this.isZoomed) {
-      return;
-    }
-    this.isZoomed = false;
+    // We need to reset the zoom domain unconditionally, as the data or the
+    // smoothing may have updated, such that we are not longer fully zoomed out.
     let xScale = this.xScale() as any;
-    xScale._domainMin = null;
```
Comment (taion) [inline tensorflow/tensorboard/components/vz_line_chart/dragZoomInteraction.ts]: No hits for these anywhere in the code base, so I assume this was just cruft.

**Comment by taion** [comment]
Any update here?

File: tensorflow/tensorboard/components/vz_line_chart/dragZoomInteraction.ts (hunk 1 above)
Comment (taion) [inline tensorflow/tensorboard/components/vz_line_chart/dragZoomInteraction.ts]: oops, never mind

**Comment by martinwicke** [comment]
Chi, can you review this?

**Comment by chihuahua** [comment]
@taion, I am very sorry for having neglected this PR. tensorboard has at this point into a separate repository. Could you please recreate the PR for the tensorflow/tensorboard repo and reference issue tensorflow/tensorboard#36? @dandelionmane can review the PR. We promise to be more responsive.

**Pull request closed without merging**